conduct.md

```diff
@@ -132,7 +132,13 @@ TODO: Write an introductory paragraph.
 
 ### Infractions
 
-TODO: Write an introductory paragraph.
+Infractions are violations of the class conduct guide as specified by the unacceptable conduct sections.
+
+When a violation is committed, the violation is added to a point total and a penalty is given according to the total violation count received by the student.  The point total is counted throughout the semester and is not removed until the class is over for the semester.
+
```

Comment:
> Could it be noted in this section that violations of the Honor Code will be enforced by the Honor Committee and not by this guideline?

Since the intro paragraph is just mentioning the basics of violations, would putting the honor code statement at the end of the levels list sound better?  I just feel that putting the more detailed info on violations and how individual types are handled would go better at the end.

Reply:
I think it's better to establish in the introduction that Honor Code violations will be enforced separately from the infraction guidelines.

Reply:
Alright, I included it in the intro.